### DISCUSSION TITLE
Stabilization in Correspondence Material Models

### DISCUSSION THREAD

**Discussion by ppandit95** [peridigm/peridigm#196] [General]
Dear Peridigm Community

I was looking into the stabilization carried out for Correspondence Models which I believe is implemented in Peridigm but I would like to know whether it is actually implemented or not.

Regards
Pushkar

**Comment by djlittl** [comment]
Yes, it is.  The magnitude of the stabilization is controlled with the "Hourglass Coefficient" entry in the input deck.  So any of the tests that have this term in the input deck is using stabilization.
